### ISSUE TITLE
Native JS Fancybox when?

### ISSUE THREAD

**Issue by avianion** [fancyapps/fancybox#2588]
I really like Fancybox, and the functionality that it provides - however I don't like having to include jQuery just to run fancybox - which is what I have to do. 

Is there a planned version that doesn't depend on Jquery?

**Comment by Cokaps** [comment]
Yeah I agree with you. I don't like using jQuery to run the Fancybox only. Because:

- jQuery is a legacy library. Standardized features like querySelector, CSS animations, and fetch make many of its features obsolete.
- Using jQuery over standard features is a waste of bandwidth.
- Because jQuery is so bloated, using jQuery often means using jQuery for everything, which means you learn less about standard web development.
- jQuery has fallen out of fashion, and full frameworks (React, Angular, Vue) are more popular.
- jQuery's cross-browser support can be substituted with the few polyfills you actually need. This also makes it easier to update when features become better supported.

**Comment by Zauberbutter** [comment]
@fancyapps Are there more accessibility features integrated or is there already a possibility to contribute?

**Comment by fancyapps** [comment]
Hi, 

See https://github.com/fancyapps/fancybox/issues/1346#issuecomment-853844350

@Zauberbutter There are some accessibility improvements, maybe you can check out the preview and give a feedback.
